library/src/main/java/link/fls/swipestack/SwipeStack.java

```diff
@@ -438,6 +446,16 @@ public void resetStack() {
          * Called when the last view has been dismissed.
          */
         void onStackEmpty();
+
+        /**
+         * Called when there is a click action.
+         */
+        void onClick();
```

Comment:
Might be a good idea to update the javadoc on the listener, adding mentions of these new methods.